pontoon/messaging/templates/messaging/badge_notification.jinja

```diff
@@ -0,0 +1,5 @@
+You have gained a new badge level!
+<br>
+{{ badge }}: Level {{ level }}
+<br>
+You can view this badge on your <a href={{ profile_href }}> profile page </a>.
```

Comment:
Mind the missing empty line. I suggest you tweak your editor settings to add them automatically.